test/github_pullrequest_remote_task_annotations_test.go

```diff
@@ -28,11 +29,20 @@ func TestGithubPullRequestRemoteTaskAnnotations(t *testing.T) {
 	runcnx, opts, ghcnx, err := tgithub.Setup(ctx, false)
 	assert.NilError(t, err)
 
+	remoteTaskURL := "https://raw.githubusercontent.com/chmouel/scratchmyback/8c78776d6ebcd052f8ee603b3d1aa20711f87a1d/.other-tasks/task-remote.yaml"
+	if os.Getenv("TEST_GITHUB_PRIVATE_TASK_URL") != "" {
+		remoteTaskURL = os.Getenv("TEST_GITHUB_PRIVATE_TASK_URL")
```

Comment:
hopefully this would work when we get merged with the change to kind-e2e-tests.yaml, the one we setup is a private url which should take fetch via the api thing